cerberus/client.py

```diff
@@ -69,12 +69,19 @@ def __init__(self, cerberus_url, username=None, password=None,
         self.HEADERS['X-Cerberus-Client'] = 'CerberusPythonClient/' + CLIENT_VERSION
 
     # noinspection PyMethodMayBeStatic
-    def _add_slash(self, string=None):
+    def _add_trailing_slash(self, string=None):
         """ if a string doesn't end in a '/' add one """
         if not str.endswith(string, '/'):
             return str.join('', [string, '/'])
         return str(string)
 
+    # noinspection PyMethodMayBeStatic
+    def _remove_trailing_slash(self, string=None):
```

Comment:
I know that the string=None part is borrowed from the `_add_trailing_slash` function but I don't know if having a default value makes sense.

Reply:
Also can you add some unit tests in test_cerberus_client.py?

Reply:
We made some changes, please re-review them :)

Reply:
I added some comments for minor changes but otherwise it looks good! And thanks for reformatting.